scripts/test-examples

```diff
@@ -61,11 +61,16 @@ function bootBroccoli() {
       examplesEnv.EXAMPLES = 1;
 
       var broccoli = childProcess.exec('node ' + broccoliBin + ' serve', { env: examplesEnv });
+      var broccoliOutput = "";
 
       broccoli.stdout.on('data', function (data) {
-        log('broccoli', data);
```

Comment:
Well, the `end` event is certain. That's based on [this](https://nodejs.org/api/stream.html#stream_event_data) and [this](https://nodejs.org/api/stream.html#stream_event_end). I believe it makes sense to process it when we have the complete output.

In terms of where to put the logging, i'm not sure if it matters on either `data` or `end`